app/models/attendee.js

```diff
@@ -35,6 +35,7 @@ export default ModelBase.extend({
   linkedin             : attr('string'),
   gender               : attr('string'),
   ageGroup             : attr('string'),
+  profileImage         : attr('number'),
```

Comment:
You cannot add fields here until you add this field in server as well

Reply:
@MeghalBisht 

 
> You can join the weekly meetings on Tues & Thursday
Where it is hosted?